dlf/plugins/toc/locallang.xml

```diff
@@ -19,12 +19,18 @@
 			<label index="tt_content.pi_flexform.excludeOther">Show only documents from the selected page</label>
 			<label index="tt_content.pi_flexform.targetPid">Target page (with "DLF: Page View" plugin)</label>
 			<label index="tt_content.pi_flexform.templateFile">Template file</label>
+			<label index="tt_content.pi_flexform.basketButton">Show basket button</label>
+			<label index="tt_content.pi_flexform.targetBasket">Target page (with "DLF: Basket" plugin)</label>
```

Comment:
You added those options to the flexform of the `pageview` plugin, but this is the localization file of the `toc` plugin. Could you please check this for consistency?

Reply:
The toc plugin has this configuration options too.